packages/hooks/src/useTransferMessageBetweenWalletAndTradingApp.ts

```diff
@@ -0,0 +1,74 @@
+import useExchangeRate from './useExchangeRate';
+import useWalletTransfer from './useWalletTransfer';
+
+type TTransferMessageBetweenWalletAndTradingApp =
+    | {
+          code: 'WalletToTradingAppDailyLimit';
+          is_first_transfer: boolean;
+          limit: number;
+          currency: string;
+          type: 'error' | 'info' | 'success';
+      }
+    | {
+          code: 'DemoWalletToTradingAppDailyLimit';
+          is_first_transfer: boolean;
+          limit: number;
+          currency: string;
+          type: 'error' | 'info' | 'success';
+      };
+
+const tradingAccountMapper = (account_type: string, currency_type: string): string => {
+    if (account_type === 'trading')
```

Comment:
@aum-deriv please, add {} after 1st if, because it's hard to understand as for me. But if it's ok to you, you can ignore it